Jint/Options.cs

```diff
@@ -43,6 +44,17 @@ public Options Strict(bool strict = true)
         }
 
         /// <summary>
+        /// Uses DateTimeKind.Local when updating native value
+        /// </summary>
+        /// <param name="clrSetLocalDateTimeKind"></param>
+        /// <returns></returns>
+        public Options ClrSetLocalDateTimeKind(bool clrSetLocalDateTimeKind = true)
```

Comment:
Can we change it to `UseLocalTimeZone(bool useLocalTimeZone = true)` ?

Reply:
Hum, then it wouldn't be obvious that's for DateTime.

`UseLocalDateTime(bool useLocalDateTime = true)`